docs/security/host-certs.md

```diff
@@ -205,6 +205,56 @@ For instructions on how to request a host or service certificate using the Web
 interface please see the [user guide maintained by
 the OIM development team](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
 
+Requesting Host Certificate for HTCondor-CE Using Let's Encrypt
+---------------------------------------------------------------
+
+As an alternative to the above options, the Let's Encrypt software can be used to obtain host certificates.
```

Comment:
Let's make `Let's Encrypt` a link to their official page and mention that the host certificates are signed by the Let's Encrypt CA.

Reply:
Great idea.